### DISCUSSION TITLE
disqus not working

### DISCUSSION THREAD

**Discussion by safinakib** [cotes2020/jekyll-theme-chirpy#1179] [Q&A]
#this is my config
comments:
  active: disqus 
  disqus:
    shortname: runit1-me 

#it says "
We were unable to load Disqus. If you are a moderator please see our [troubleshooting guide](https://docs.disqus.com/help/83/)."

**Comment by ciphermenial** [comment]
See troubleshooting guide. The issue here is not with Chirpy.
